app/src/main/java/com/amaze/filemanager/ui/activities/AboutActivity.java

```diff
@@ -79,15 +81,15 @@
   @Override
   public void onCreate(@Nullable Bundle savedInstanceState) {
     super.onCreate(savedInstanceState);
-
-    if (getAppTheme().equals(AppTheme.DARK)) {
-      setTheme(R.style.aboutDark);
-    } else if (getAppTheme().equals(AppTheme.BLACK)) {
-      setTheme(R.style.aboutBlack);
-    } else {
-      setTheme(R.style.aboutLight);
+    if (Build.VERSION.SDK_INT < Build.VERSION_CODES.LOLLIPOP) {
```

Comment:
And for versions older that lollipop?

Reply:
I have checked the official documentation and found that the color of the navigation bar cannot be manipulated for sdk versions before Lollipop, the closest you can get is making the bar translucent